<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MDLT011" name="UnbreakableTestTrial" tests="2" failures="2" timestamp="21 Jul 2020 14:57:43 GMT" time="12.045" errors="0">
  <testcase name="afterMethod" time="8.891" classname="UnbreakableTestTrial">
    <failure type="java.lang.AssertionError" message="A detailed report will be created at &#039;D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report&#039;. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

4 check(s) in &#039;UnbreakableTestTrial&#039; found the following difference(s):
Test &#039;beforeMethod&#039; has 2 difference(s) in 4 state(s):
00 resulted in:
	Metadata Differences:
		
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[1]&#039;:
		id: expected=&quot;username123&quot;, actual=&quot;username123456&quot;
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[2]&#039;:
		id: expected=&quot;password&quot;, actual=&quot;password123&quot;
01 resulted in:
	Metadata Differences:
		
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[1]&#039;:
		id: expected=&quot;username123&quot;, actual=&quot;username123456&quot;
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[2]&#039;:
		id: expected=&quot;password&quot;, actual=&quot;password123&quot;
02 resulted in:
	Metadata Differences:
		
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[1]&#039;:
		id: expected=&quot;username123&quot;, actual=&quot;username123456&quot;
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[2]&#039;:
		id: expected=&quot;password&quot;, actual=&quot;password123&quot;
03 resulted in:
	Metadata Differences:
		
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[1]&#039;:
		id: expected=&quot;username123&quot;, actual=&quot;username123456&quot;
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[2]&#039;:
		id: expected=&quot;password&quot;, actual=&quot;password123&quot;">
      <![CDATA[java.lang.AssertionError: A detailed report will be created at 'D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report'. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

4 check(s) in 'UnbreakableTestTrial' found the following difference(s):
Test 'beforeMethod' has 2 difference(s) in 4 state(s):
00 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
01 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
02 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
03 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
at de.retest.recheck.RecheckImpl.capTest(RecheckImpl.java:188)
at de.retest.recheck.RecheckImpl.cap(RecheckImpl.java:200)
at de.retest.web.selenium.AutocheckingRecheckDriver.cap(AutocheckingRecheckDriver.java:88)
at de.retest.web.selenium.AutocheckingRecheckDriver.quit(AutocheckingRecheckDriver.java:108)
at UnbreakableTestTrial.afterMethod(UnbreakableTestTrial.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- afterMethod -->
  <testcase name="unbreakableTest" time="3.154" classname="UnbreakableTestTrial">
    <failure type="java.lang.AssertionError" message="A detailed report will be created at &#039;D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report&#039;. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

1 check(s) in &#039;UnbreakableTestTrial&#039; found the following difference(s):
Test &#039;beforeMethod&#039; has 2 difference(s) in 1 state(s):
trial resulted in:
	Metadata Differences:
		
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[1]&#039;:
		id: expected=&quot;username123&quot;, actual=&quot;username123456&quot;
	input at &#039;html[1]/body[1]/main[1]/form[1]/input[2]&#039;:
		id: expected=&quot;password&quot;, actual=&quot;password123&quot;">
      <![CDATA[java.lang.AssertionError: A detailed report will be created at 'D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report'. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

1 check(s) in 'UnbreakableTestTrial' found the following difference(s):
Test 'beforeMethod' has 2 difference(s) in 1 state(s):
trial resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
at de.retest.recheck.RecheckImpl.capTest(RecheckImpl.java:188)
at UnbreakableTestTrial.unbreakableTest(UnbreakableTestTrial.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- unbreakableTest -->
</testsuite> <!-- UnbreakableTestTrial -->
